drivers/volume/portworx/portworx.go

```diff
@@ -1572,18 +1572,29 @@ func (d *portworx) CollectDiags(n node.Node) error {
 		return err
 	}
 
+	opts := node.ConnectionOpts{
```

Comment:
Why did you remove Sudo: true from here, it was required for collecting diags on some of the systems. It was added in PTX-1825

Reply:
@nikolaypopov , Added "Sudo: true" back.